### PR TITLE
update instructions for fiscal year

### PR DESCRIPTION
**Pull request by nichelle-hall**
## Description of the change

Updates instructions to help the user correctly upload for fiscal-year metrics. 

Screenshot 2023-09-19 at 12 03 00 PM

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/DataUpload/InstructionsTemplate.tsx [hunk 1]
```diff
@@ -349,6 +349,13 @@ export const GeneralInstructions: React.FC<
         value should be numeric and contain no other characters (commas are
         allowed, e.g. 1,000).
       </p>
+      <p>
+        If you are uploading for a fiscal year, the year you provide in the year
+        column represents the end year of the record. For example, if you would
+        like to report an annual metric for a fiscal year that started July 1,
```
Comment (morden35) [inline publisher/src/components/DataUpload/InstructionsTemplate.tsx:355]: nit: report -> record

**Review by morden35 — APPROVED** [review]
lgtm! 1 nit in-line

File: publisher/src/components/DataUpload/InstructionsTemplate.tsx (hunk 1 above)
Comment (mxosman) [reply publisher/src/components/DataUpload/InstructionsTemplate.tsx:355]: Nice catch!!

**Review by mxosman — APPROVED** [review]
LGTM too!

**Review by morden35 — APPROVED** [review]

**Pull request merged**